src/telliot_feeds/reporters/tips/__init__.py

```diff
@@ -63,3 +63,4 @@ def add_multicall_support(
 
 CATALOG_QUERY_IDS = {query_catalog._entries[tag].query.query_id: tag for tag in query_catalog._entries}
 CATALOG_QUERY_DATA = {query_catalog._entries[tag].query.query_data: tag for tag in query_catalog._entries}
+TYPES_WITH_GENERIC_SOURCE = ["MimicryMacroMarketMashup", "MimicryCollectionStat"]
```

Comment:
I'd add a note/comment about what this is for, or why it's needed for ppl who aren't as familiar w/ the codebase, as `TYPES_WITH_GENERIC_SOURCE` as a name is kind of vague.